<launch>
<master auto="start"/>
<!--Remaps the /map topic produced by map_server to /ground_truth  -->
<!-- <remap from="/map" to ="/ground_truth" />  -->

<rosparam file="$(find random_walk)/config/config.yaml" />

<node pkg="random_walk" name="add_robots" type="add_robot.py" output="screen"></node>
<node pkg="random_walk" name="add_slam" type="add_slam.py" output="screen"></node>
<node pkg="random_walk" name="add_mapmerge" type="add_mapmerge.py" output="screen"></node>

<!--Pass the ground truth map to the as an argument  -->
<arg name="map_file" default="$(find random_walk)/stage_results/final.yaml"/>


<!--/home/nikos/catkin_ws/src/random_walk/world/willow-four-erratics.world -->
<!-- Starts the map_server with the ground truth map -->
<node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

<!-- Rviz node -->
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find random_walk)/rviz/multi_turtlebot3_slam.rviz" />

<!-- Starts the Stage Simulation -->
<!--  <node name="Stage" pkg="stage_ros" type="stageros" args="/home/nikos/catkin_ws/src/random_walk/world/test.world">
    </node> -->

</launch>